### DISCUSSION TITLE
How to configure system tray

### DISCUSSION THREAD

**Discussion by Citronax** [eythaann/Seelen-UI#334] [General]
How can I configure the system tray to permanently display "Safely Remove Hardware and Eject Media?"

Many thanks for any help provided.

**Comment by eythaann** [comment]
Currently, there's no way to add/pin tray items on the toolbar, but I think there's an existing ticket created to implement that as a feature.

**Reply by Citronax** [reply]
Thank you for the prompt response, most appreciated.
